tests/testthat/test-shinytest2-teal_slices.R

```diff
@@ -99,3 +99,25 @@ testthat::test_that("e2e: teal_slices filters are initialized when module specif
   )
   app$stop()
 })
+
+testthat::test_that("e2e: Set numeric filter state", {
```

Comment:
I think this test is mainly testing the `TealAppDriver$set_active_filter_selection` method. In this case I would think that it's better placed inside a new file dedicated to test the public methods from the class.
We can still test some business logic that uses `set_active_filter_selection`. For example, setting the filter state changes the filter state in another module (when module_specific is FALSE) and shared module (when module_specific is TRUE)

Also, the test `e2e: teal_slices filters are initialized when module specific filters are created` can remove the last part where we use `set_active_filter_selection`. It can stop after expecting the initialized values as suggested by the test desc.

Reply:
The idea was to test the filter panel using the `shinytest2` way. But you're right, it only tests the function.

I'd say remove the test completely as well as that last part of the other test

Reply:
Re-include once we start to test `TealAppDriver`

Reply:
Done with 3346cc7